hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/InvocationMonitor.java

```diff
@@ -501,7 +505,7 @@ private void sendOpControlPacket(Address address, OperationControl opControl) {
                 Packet packet = new Packet(serializationService.toBytes(opControl))
                         .setPacketType(Packet.Type.OPERATION)
                         .raiseFlags(FLAG_OP_CONTROL | FLAG_URGENT);
-                nodeEngine.getNode().getConnectionManager().transmit(packet, address);
+                connectionManager.transmit(packet, address);
```

Comment:
Normally there is an ordering problem. The connection manager is made after the operation service is made, so you would get a null as connectionManager.